README.md

```diff
@@ -263,47 +254,13 @@ TDR datasets and snapshots is granted by adding the registered service accounts
 to dedicated SAM groups (an extension of a Google group). This must be
 performed manually by someone with administrator access to that SAM group. For
 non-production instances of TDR, the indexer service account should be added to
-the group `azul-dev`. The only members of this group should be service
-accounts belonging to non-production deployments of Azul and burner accounts.
+the group `azul-dev`.
 
 A member of the `azul-dev` group has read access to TDR. An *administrator* of
 this group can add other accounts to it, and optionally make them
 administrators, too. Before any account can be added to a group, it needs to be
-registered with SAM. While `make deploy` does this automatically for the
-deployment's service account, for your burner you must follow the steps below:
-
-
-1. Log into Google Cloud by running
-
-    ```
-    gcloud auth login
-    ```
-
-    A browser window opens to complete the authentication flow interactively.
-    When being prompted, select your burner.
-
-    For more information refer to the Google authorization
-    [documentation](https://cloud.google.com/sdk/docs/authorizing).
-
-2. Register your burner with SAM. Run
-
-    ```
-    (account="$(gcloud config get-value account)"
-    token="$(gcloud auth --account $account print-access-token)"
-    curl $AZUL_SAM_SERVICE_URL/register/user/v1  -d "" -H "Authorization: Bearer $token")
-    ```
-
-3. Ask an administrator of the `azul-dev` group to add your burner to the
-   group. The best way to reach an administrator is via the `#team-boardwalk`
-   channel on Slack. Also, ask for a link to the group and note it in your
-   records.
-
-4. If you've already attempted to create your deployment via `make deploy`,
-   visit the link, sign in as your burner and add your deployment's service
-   account to the group. Run `make deploy` again.
-
-For production, use the same procedure, but substitute `azul-dev` with
-`azul-prod` and "burner" with "account".
```

Comment:
I have a suspicion that this section still applies, just to our personal developer accounts rather than our burner accounts. We still need to authenticate with Terra (and therefore need an account registered with SAM); we just won't be using our burners for this purpose any longer. I think the only way I could be wrong is if developer account registration is already documented somewhere else.